src/disco/components/Addon/index.js

```diff
@@ -226,25 +231,32 @@ export class AddonBase extends React.Component {
   };
 
   clickHeadingLink = (e) => {
-    const { type, name, _tracking } = this.props;
+    const { addon, _tracking } = this.props;
 
     if (e.target.nodeName.toLowerCase() === 'a') {
       _tracking.sendEvent({
-        action: getAddonTypeForTracking(type),
+        action: getAddonTypeForTracking(addon.type),
```

Comment:
🔥 I hope this is correct 😀 I'm sure it is. The old reliance on spreads was absolutely bonkers.